### ISSUE TITLE
Module '"pg"' has no exported member 'escapeLiteral'

### ISSUE THREAD

**Issue by cesco69** [brianc/node-postgres#3130]
```ts
import { escapeLiteral  } from 'pg';
```

> Module '"pg"' has no exported member 'escapeLiteral'

```json
{
  "pg": "^8.11.3"
}
```

**Comment by thesauri** [comment]
The type seems to be missing from the corresponding types library: https://github.com/brianc/node-postgres/issues/3099

The fix hasn't been merged yet: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68112

**Comment by cesco69** [comment]
ok, this is a workaround for the users come here

```ts
// types.d.ts
declare module 'pg' {
    export function escapeIdentifier(str: string): string;
    export function escapeLiteral(str: string): string;
}
```

**Comment by gabegorelick** [comment]
This is fixed in the latest version of `@types/pg` (8.11.0). I think this issue can be closed.
